cves/CVE-2017-14058.yml

```diff
@@ -64,7 +64,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: 
+  a read function in ffmpeg does not restrict reload attempts 
```

Comment:
Capitalize the first word of the sentence.

Reply:
Thanks fixed